spec/collectionspace/converter/publicart/acquisition_spec.rb

```diff
@@ -40,69 +40,44 @@
     end
 
     context 'fields overridden by publicart' do
-      context 'when local place' do
+      context 'authority/vocab fields' do
         [
-          "/document/#{p}/owner",
-          "/document/#{p}/acquisitionSource",
```

Comment:
@Paulmulonzia -- need to include full xpath for repeatable fields and field groups

Reply:
I must have missed that somehow.